docs/website/data/integrations.yaml

```diff
@@ -1415,8 +1415,19 @@ integrations:
       - https://github.com/rond-authz/rond
     inventors:
       - mia-platform
+    tutorials: 
+      - https://github.com/rond-authz/example
+    videos:
+      - title: "Rönd - The Open Source K8s sidecar that defines security policies over your APIs"
+        speakers:
+          - name: Federico Maggi
+            organization: mia-platform
+        link: https://www.youtube.com/watch?v=ubT31NtHV8w
     blogs:
-      - https://blog.mia-platform.eu/en/how-why-adopted-role-based-access-control-rbac
+      - title: Announcing Rönd, a new open-source Security Enforcement over your APIs
```

Comment:
The structure isn't right here, I think -- try dropping the titles. [It expects only a list of URLs](https://sourcegraph.com/github.com/open-policy-agent/opa/-/blob/docs/website/layouts/docs/ecosystem.html.html?L218-220&subtree=true)